index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile Version Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,400;0,600;1,500&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+  <main class="container">
+    <section class="introduction">
+      <header class="main-header">
+        <!-- flex-container -->
+        <div class="toolbar"> 
+          <!-- flex-items -->
+            <div class="three time"><img src="./images/Time.png" alt=""></div>
+            <nav>
+              <div class="three"><img src="./images/Cellular.png" alt=""></div>
+              <div class="three"><img src="./images/Wifi.png" alt=""></div>
+              <div class="three"><img src="./images/Battery.png" alt=""></div>
+            </nav>
+      </div>
```

Comment:
Kindly, the phone information bar is not necessary to emulate it in your portfolio